src/include/rbd/librbd.hpp

```diff
@@ -22,6 +22,7 @@
 #include <vector>
 #include "../rados/buffer.h"
 #include "../rados/librados.hpp"
+#include "../utime.h"
```

Comment:
"utime.h" isn't part of the public API -- just convert to "time_t" for public API